orcid-core/src/main/java/org/orcid/core/manager/impl/OrcidJaxbCopyManagerImpl.java

```diff
@@ -342,9 +342,14 @@ private void copyUpdatedEmails(ContactDetails existingContactDetails, ContactDet
 				}
 			}
 		}
+		//Set primary = false for each remaining new email.
 		if(updatedContactDetails != null) {
-			allEmails.addAll(updatedContactDetails.getEmail());
+			for(Email newEmail : updatedContactDetails.getEmail()) {
```

Comment:
@rcpeters This code is called only when updating the bio. Is there a particular flow that you have in mind? I can try.